### DISCUSSION TITLE
Awesome work

### DISCUSSION THREAD

**Discussion by Abraka** [kubesphere/kubesphere#4182] [General]
Just login to say, that you did amazing work here. Especialy I like graphic of the interface. Awesome!

**Comment by yngfoxx** [comment]
I was looking for a place to review this amazing software, if I can give all my money to Kubesphere even though it is free I will for real do it, this software is just mind blowing - I don't have enough words to use to express this but to say GREAT JOB 👍 and thank you for saving me a whole month worth of time.

**Reply by rayzhou2017** [reply]
Thanks @yngfoxx ! It would be helpful to tell more and more people about KubeSphere. BTW, we are looking for local partner, so if anyone is interested please don't hesitate to contact us.
